homeassistant/components/thermostat/zwave.py

```diff
@@ -52,6 +52,10 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
                 _LOGGER.debug("Remotec ZXT-120 Zwave Thermostat, ignoring")
                 return
 
+    if value.command_class != COMMAND_CLASS_SENSOR_MULTILEVEL and \
+       value.command_class != COMMAND_CLASS_THERMOSTAT_SETPOINT:
+        return
```

Comment:
Wait, now you're checking if 1 value is equal to 2 different values. I'm assuming those 2 values are not the same so this test will _always_ be false.